config.plist/alder-lake.md

```diff
@@ -0,0 +1,793 @@
+| Limit | Number | 0 |
+| Mask | Data | <> |
+| MaxKernel | String | (this value should be empty, do not add this string to your config) |
+| MinKernel | String | 19.0.0 |
+| Replace | Data | C3 |
+| ReplaceMask | Data | <> |
+| Skip | Number | 0 |
+
+:::
+
+### Quirks
+
+::: tip Info
+
+Settings relating to the kernel, for us we'll be enabling the following:
+
+| Quirk | Enabled | Comment |
+| :--- | :--- | :--- |
+| AppleXcpmCfgLock | YES | Disable this quirk if `CFG Lock` is disabled in the BIOS. |
+| DisableIoMapper | YES | Disable this quirk if `VT-d` is disabled in the BIOS. |
+| LapicKernelPanic | NO | HP Machines will require this quirk. |
+| PanicNoKextDump | YES | |
+| PowerTimeoutKernelPanic | YES | |
+| ProvideCurrentCpuInfo | YES | |
+| XhciPortLimit | YES | Disable if running macOS 11.3+. |
+
+:::
+
+::: details More in-depth Info
+
+* **AppleCpuPmCfgLock**: NO
+  * Only needed when CFG-Lock can't be disabled in BIOS
+  * Only applicable for Ivy Bridge and older
+    * Note: Broadwell and older require this when running 10.10 or older
+* **AppleXcpmCfgLock**: YES
+  * Only needed when CFG-Lock can't be disabled in BIOS
+  * Only applicable for Haswell and newer
+    * Note: Ivy Bridge-E is also included as it's XCPM capable
+* **CustomSMBIOSGuid**: NO
+  * Performs GUID patching for UpdateSMBIOSMode set to `Custom`. Usually relevant for Dell laptops
+  * Enabling this quirk with UpdateSMBIOSMode Custom mode can also disable SMBIOS injection into "non-Apple" OSes however we do not endorse this method as it breaks Bootcamp compatibility. Use at your own risk
+* **DisableIoMapper**: YES
+  * Needed to get around VT-D if either unable to disable in BIOS or needed for other operating systems, much better alternative to `dart=0` as SIP can stay on in Catalina
+* **DisableLinkeditJettison**: YES
+  * Allows Lilu and others to have more reliable performance without `keepsyms=1`
+* **DisableRtcChecksum**: NO
+  * Prevents AppleRTC from writing to primary checksum (0x58-0x59), required for users who either receive BIOS reset or are sent into Safe mode after reboot/shutdown
+* **ExtendBTFeatureFlags** NO
+  * Helpful for those having continuity issues with non-Apple/non-Fenvi cards
+* **LapicKernelPanic**: NO
+  * Disables kernel panic on AP core lapic interrupt, generally needed for HP systems. Clover equivalent is `Kernel LAPIC`
+* **LegacyCommpage**: NO
+  * Resolves SSSE3 requirement for 64 Bit CPUs in macOS, mainly relevant for 64-Bit Pentium 4 CPUs(ie. Prescott)
+* **PanicNoKextDump**: YES
+  * Allows for reading kernel panics logs when kernel panics occur
+* **PowerTimeoutKernelPanic**: YES
+  * Helps fix kernel panics relating to power changes with Apple drivers in macOS Catalina, most notably with digital audio.
+* **SetApfsTrimTimeout**: `-1`
+  * Sets trim timeout in microseconds for APFS filesystems on SSDs, only applicable for macOS 10.14 and newer with problematic SSDs.
+* **XhciPortLimit**: YES
```

Comment:
We should not be recommending it on anymore

Reply:
Resolved in 5889bfa